src/Ledger/Certs/Properties/PoVLemmas.lagda

```diff
@@ -0,0 +1,245 @@
+            ≡⟨ indexedSumᵛ'-∪ (rewards ∣ dom wdrlsCC ᶜ) (rewards ∣ dom wdrlsCC)
+                              (disjoint-sym res-ex-disjoint) ⟩
+          getCoin (rewards ∣ dom wdrlsCC ᶜ) + getCoin (rewards ∣ dom wdrlsCC )
+            ≡⟨ cong (getCoin (rewards ∣ dom wdrlsCC ᶜ) +_)
+               ( getCoin-cong (rewards ∣ dom wdrlsCC) wdrlsCC (res-subset{m = rewards} wdrlsCC⊆rwds) ) ⟩
+          getCoin (rewards ∣ dom wdrlsCC ᶜ) + getCoin wdrlsCC
+            ≡⟨ cong (getCoin (rewards ∣ dom wdrlsCC ᶜ) +_) (≡ᵉ-getCoinˢ (wdrls ˢ) (injOn wdrls validNetId)) ⟩
+          getCoin (rewards ∣ dom wdrlsCC ᶜ) + getCoin wdrls
+            ≡˘⟨ cong (_+ getCoin wdrls)
+                ( begin
+                  getCoin (zeroMap ∪ˡ rewards)
+                    ≡⟨ ≡ᵉ-getCoin (zeroMap ∪ˡ rewards) (zeroMap ∪ˡ (rewards ∣ dom zeroMap ᶜ))
+                                  (res-decomp zeroMap rewards) ⟩
+                  getCoin (zeroMap ∪ˡ (rewards ∣ dom zeroMap ᶜ))
+                    ≡⟨ indexedSumᵛ'-∪ zeroMap (rewards ∣ dom zeroMap ᶜ)
+                                      (disjoint-sym res-comp-dom) ⟩
+                  getCoin zeroMap + getCoin (rewards ∣ dom zeroMap ᶜ)
+                    ≡⟨ cong (λ u → u + getCoin (rewards ∣ dom zeroMap ᶜ)) sumConstZero ⟩
+                  0 + getCoin (rewards ∣ (dom zeroMap) ᶜ)
+                    ≡⟨ +-identityˡ (getCoin (rewards ∣ dom zeroMap ᶜ)) ⟩
+                  getCoin (rewards ∣ dom zeroMap ᶜ)
+                    ≡⟨ ≡ᵉ-getCoin (rewards ∣ dom zeroMap ᶜ) (rewards ∣ dom wdrlsCC ᶜ)
+                       ( res-comp-cong
+                         ( ⊆-Transitive (proj₁ constMap-dom) (proj₂ dom-mapˡ≡map-dom)
+                         , ⊆-Transitive (proj₁ dom-mapˡ≡map-dom) (proj₂ constMap-dom) ) ) ⟩
+                  getCoin (rewards ∣ dom wdrlsCC ᶜ)
+                    ∎ ) ⟩
+          getCoin (zeroMap ∪ˡ rewards) + getCoin wdrls
+            ∎
+\end{code}
+  \end{itemize}
+\end{lemma}
+
+
+\begin{lemma}[%
+  \LedgerMod{\CertsPoVL.lagda}{\AgdaModule{\CertsPoVL{}}}: iteration of \CERT{} rule preserves value%
+  ]
+  \begin{itemize}
+    \item \textit{Informally}.
+      Let \ab{l} be a list of \DCert{}s, and let \ab{s₁}, \ab{sₙ} be \CertState{}s such
+      that, starting with \ab{s₁} and successively applying the \CERT{} rule to
+      with \DCert{}s from the list \ab{l}, we obtain \ab{sₙ}.
+      Then, the value of \ab{s₁} is equal to the value of \ab{sₙ}.      
+    \item \textit{Formally}.
+\begin{code}
+    sts-pov : {Γ : CertEnv} {s₁ sₙ : CertState}
+      → ReflexiveTransitiveClosure {sts = _⊢_⇀⦇_,CERT⦈_} Γ s₁ l sₙ
+      → getCoin s₁ ≡ getCoin sₙ
+\end{code}
+    \item \textit{Proof}. See the
+      \LedgerMod{\CertsPoVL.lagda}{\AgdaModule{\CertsPoVL{}}}
+      module in the \href{\repourl}{formal ledger repository}.
+\begin{code}[hide]
+    -- Proof.
+    sts-pov (BS-base Id-nop) = refl
+    sts-pov (BS-ind x xs) = trans (CERT-pov x) (sts-pov xs)
+\end{code}
+  \end{itemize}
+\end{lemma}
```

Comment:
How is this lemma different from the theorem in Ledger.Certs.Properties.PoV? (Since CERTS is defined as the "reflexive-transitive closure" of CERTBASE)

Reply:
Actually, `CERTS` is the "refl-trans closure" of `CERT` (with `CERTSBASE` as the "base step" ...though it's not really a base step).  So, the proof of `CERTS-pov` uses both the `sts-pov` and the `CERTBASE-pov` lemmas.  If you think we should inline `sts-pov` and not present it as a separate lemma, that's probably a good idea.  However, I don't want to refactor proofs in this PR.